docs/how-to-use-customized-configuration.md

```diff
@@ -1,294 +0,0 @@
-              ...
-          ```
-       * ScalarDL Auditor
-         ```yaml
-         auditor:
-           auditorProperties: |
-             ...
-             scalar.db.username={{ default .Env.SCALAR_DB_USERNAME "" }}
-             scalar.db.password={{ default .Env.SCALAR_DB_PASSWORD "" }}
-             ...
-         ```
-       * ScalarDL Schema Loader
-         ```yaml
-         schemaLoading:
-           databaseProperties: |
-             ...
-             scalar.db.username={{ default .Env.SCALAR_DB_USERNAME "" }}
-             scalar.db.password={{ default .Env.SCALAR_DB_PASSWORD "" }}
-             ...
-         ```
-
-1. Create a `Secret` resource that includes credentials.  
-   You need to specify the environment variable name as keys of the `Secret`.
-   * Example
-     ```console
-     kubectl create secret generic scalardb-credentials-secret \
-       --from-literal=SCALAR_DB_USERNAME=postgres \
-       --from-literal=SCALAR_DB_PASSWORD=postgres
-     ```
-
-1. Set the `Secret` name to the following keys in the custom values file.  
-   * Keys
-     * `scalardb.secretName` (ScalarDB)
-     * `ledger.secretName` (ScalarDL Ledger)
-     * `auditor.secretName` (ScalarDL Auditor)
-     * `schemaLoading.secretName` (ScalarDL Schema Loader)
-   * Example
-     * ScalarDB
-       ```yaml
-       scalardb:
-         secretName: "scalardb-credentials-secret"
-       ```
-     * ScalarDL Ledger
-       ```yaml
-       ledger:
-         secretName: "ledger-credentials-secret"
-       ```
-     * ScalarDL Auditor
-       ```yaml
-       auditor:
-         secretName: "auditor-credentials-secret"
-       ```
-     * ScalarDL Schema Loader
-       ```yaml
-       schemaLoading:
-         secretName: "schema-loader-ledger-credentials-secret"
-       ```
-
-1. Deploy Scalar products with the above custom values file.  
-   After deploying Scalar products, the Go template strings (environment variables) are replaced by the values of the `Secret`.
-   * Example
-       * Custom values file
-         ```yaml
-         scalardb:
-           databaseProperties: |
-             scalar.db.contact_points=jdbc:postgresql://postgresql-scalardb.default.svc.cluster.local:5432/postgres
-             scalar.db.username={{ default .Env.SCALAR_DB_USERNAME "" }}
-             scalar.db.password={{ default .Env.SCALAR_DB_PASSWORD "" }}
-             scalar.db.storage=jdbc
-         ```
-       * Properties file in containers
-         ```properties
-         scalar.db.contact_points=jdbc:postgresql://postgresql-scalardb.default.svc.cluster.local:5432/postgres
-         scalar.db.username=postgres
-         scalar.db.password=postgres
-         scalar.db.storage=jdbc
-         ```
-
-   Please refer to the [Getting Started with Scalar Helm Charts](./getting-started-scalar-helm-charts.md) for more details on how to use each Helm Chart.
-
-## Mount arbitrary files like key and certificate files to the container in ScalarDL Helm Charts
```

Comment:
I moved this section to [Mount key/certificate files to the pod in ScalarDL Helm Charts](https://github.com/scalar-labs/helm-charts/blob/main/docs/mount-key-and-cert-for-scalardl.md).